src/meta/processors/admin/Balancer.h

```diff
@@ -106,10 +106,6 @@ class Balancer {
 
     void finish() {
         CHECK(!lock_.try_lock());
```

Comment:
A question，Is the useful now of lock?

Reply:
Yes, we need to make sure we hold the lock whenever we want to call `finish`